TRANSLATING.md

```diff
@@ -4,24 +4,31 @@ Thanks for your interest in helping us translate [docs.astro.build](https://docs
 
 ## Getting involved
 
-### How can I help translate one of the supported languages?
-Translations all live in this GitHub repository. You can add and update them by creating a pull request. Read on to find out more!
+To get involved in our translation efforts, we **highly recommend** joining our Discord chat](https://astro.build/chat) first. This way, we can get you up to speed with our process and give you the opportunity to chat with your language's translation team.
 
-### How can I find out what needs translating?
-See our automated [Translation Status Overview issue](https://github.com/withastro/docs/issues/438) for a quick list of which pages are missing or need updating.
+Most of our internationalization decisions and discussions happen on Discord. So, it's the best way to find out which patterns and recommendations your language's translation team follows before making your own PRs. You can even become a part of the decision-making process.
 
+> Can’t access Discord? Please [open a new issue](https://github.com/withastro/docs/issues/new/choose) here on GitHub to ask any questions you may have.
+
+### How can I help translate/review one of the supported languages?
+
+Translations all live in this GitHub repository. You can add and update them by creating a pull request or reviewing pending translations in the "Pull Requests" tab. Read on to find out more!
+
+### How can I find out what needs to be reviewed/translated?
+
+To find translations pending review, you can simply filter through this repo's [Pull Requests with the i18n label](https://github.com/withastro/docs/pulls?q=is%3Apr+is%3Aopen+label%3Ai18n).
```

Comment:
In general, we try to avoid judgements like “simply” as they might imply something should be simple to do, which can be frustrating if someone is not finding it simple! I’d suggest removing that in this case.

```suggestion
To find translations pending review, you can filter through this repo's [Pull Requests with the i18n label](https://github.com/withastro/docs/pulls?q=is%3Apr+is%3Aopen+label%3Ai18n).
```

Reply:
oh, good point!